Assets/MixedRealityToolkit.Tests/PlayModeTests/TestHand.cs

```diff
@@ -61,7 +61,10 @@ public IEnumerator MoveTo(Vector3 newPosition, int numSteps = 30, bool waitForFi
         {
             Vector3 oldPosition = position;
             position = newPosition;
-            yield return PlayModeTestUtilities.MoveHandFromTo(oldPosition, newPosition, numSteps, gestureId, handedness, simulationService);
+            for (var iter = PlayModeTestUtilities.MoveHandFromTo(oldPosition, newPosition, numSteps, gestureId, handedness, simulationService); iter.MoveNext(); )
```

Comment:
I have changes these to yield each individual step, which makes it easier to investigate test failures.